### ISSUE TITLE
Incompatibility with the AuthRemoteUser Plugin: Uncaught RuntimeException: logout disabled

### ISSUE THREAD

**Issue by j0hannes** [cosmocode/dokuwiki-template-sprintdoc#81]
See title; without any configuration, this error is thrown:
```
PHP Fatal error:  Uncaught RuntimeException: logout disabled in /var/www/wikifarm/dokuwiki/inc/Menu/Item/Login.php:21
Stack trace:
#0 /var/www/wikifarm/dokuwiki/lib/tpl/sprintdoc/tpl/nav-usertools-buttons.php(14): dokuwiki\\Menu\\Item\\Login->__construct()
#1 /var/www/wikifarm/dokuwiki/lib/tpl/sprintdoc/main.php(264): include('/var/www/wikifa...')
#2 /var/www/wikifarm/dokuwiki/inc/actions.php(27): include('/var/www/wikifa...')
#3 /var/www/wikifarm/dokuwiki/doku.php(122): act_dispatch()
#4 {main}\n  thrown in /var/www/wikifarm/dokuwiki/inc/Menu/Item/Login.php on line 21
```
After re-enabling the logout option, the error disappears.

**Comment by splitbrain** [comment]
should be fixed with a80089cadd936799775013ddad0a90bf7a6a23d0

**Comment by j0hannes** [comment]
it is, thanks
